### PR TITLE
Add support key shortcut to Onboarding Modal

### PR DESCRIPTION
**Pull request by ykzts**
Add key shortcut to Onboarding Modal. left key (go to previous) & right key (go to next). This imitates the image modal.

Also, carry out refactoring to reduce the load of re-rendering.

### CODE REVIEW

File: app/javascript/mastodon/features/ui/components/onboarding_modal.js [hunk 1]
```diff
@@ -59,6 +60,7 @@ const PageTwo = ({ me }) => (
         onClearSuggestions={noop}
         onFetchSuggestions={noop}
         onSuggestionSelected={noop}
+        showSearch
```
Comment (ykzts) [inline app/javascript/mastodon/features/ui/components/onboarding_modal.js]: It is intended to avoid auto focus of the textarea.

**Comment by ykzts** [comment]
I rebased the master branch

File: app/javascript/mastodon/features/ui/components/onboarding_modal.js [hunk 2]
```diff
@@ -191,28 +214,36 @@ class OnboardingModal extends React.PureComponent {
...
   }
 
+  handlePrev = () => {
+    this.setState(({ currentIndex }) => ({
+      currentIndex: Math.max(0, currentIndex - 1),
+    }));
+  }
+
   handleNext = () => {
+    const { pages } = this.state;
```
Comment (Gargron) [inline app/javascript/mastodon/features/ui/components/onboarding_modal.js]: What is the reason for storing pages in state? It doesn't even need to be props, I think it's a constant.

Comment (ykzts) [inline app/javascript/mastodon/features/ui/components/onboarding_modal.js]: I changed `this.state.pages` to `this.pages` with 6b6baca. Is this feeling good?

**Review by Gargron — APPROVED** [review]

**Pull request merged**